lib/livebook/file_system/file.ex

```diff
@@ -49,7 +49,7 @@ defmodule Livebook.FileSystem.File do
   """
   @spec local(FileSystem.path()) :: t()
   def local(path) do
-    new(FileSystem.Local.new(), path)
+    new(Livebook.Config.local_filesystem(), path)
```

Comment:
Can we write a simple test? Something showing that two calls to local would return the filesystem? Or any property that would fail before this fix?

Reply:
I think the only property would be

```
FileSystem.File.local("whatever").file_system == Livebook.Config.local_filesystem()
```

and I'm not sure if this test adds any value.

Previously `FileSystem.File.local(...)` would always return the same file system, it's just that the file system created with `FileSystem.Local.new()` would have `default_path` set to cwd, while **the** local file system has it set to Livebook home.

Reply:
Ok, done!